source/accessibility.html.md.erb

```diff
@@ -69,9 +57,3 @@ We used manual and automated tests to look for issues such as:
 - lack of colour contrast for text, important graphics and controls
 - images not having meaningful alt text
 - problems when using assistive technologies such as screen readers and screen magnifiers
-
-## What we’re doing to improve accessibility
-
-When we publish new content, we’ll make sure our use of content meets accessibility standards.
-
-This statement was prepared on 24 September 2020. It was last updated on 24 September 2020.
```

Comment:
I've a feeling the statement last updated date also needs to stay as a line of text, in addition to it being in the header.

Reply:
I removed it on purpose as I'm trying to simplify the whole statement and it's a burden to update the date is multiple places as they can easily be missed.